graphql/version_resolver.go

```diff
@@ -305,7 +306,7 @@ func (r *versionResolver) Tasks(ctx context.Context, obj *restModel.APIVersion,
 		Limit:                          limitParam,
 		Sorts:                          taskSorts,
 		IncludeBaseTasks:               true,
-		IncludeEmptyActivation:         utility.FromBoolPtr(options.IncludeEmptyActivation),
+		IncludeInactiveTasks:           utility.FromBoolPtr(options.IncludeEmptyActivation) || utility.FromBoolPtr(options.IncludeInactiveTasks),
```

Comment:
At what point would we be removing options.IncludeEmptyActivation?

Reply:
Sometime after both the frontend and backend are deployed. I'll open a follow-up ticket for it

Reply:
Can we also add that ticket number in a comment here?